session/object.go

```diff
@@ -166,6 +166,11 @@ func (x *Object) Sign(signer neofscrypto.Signer) error {
 	return x.sign(signer, x.writeContext)
 }
 
+// SignedData returnes actual payload which would be signed, if you call [Object.Sign] method.
```

Comment:
```suggestion
// SignedData returns actual payload which would be signed, if you call [Object.Sign] method.
```

i'd also paraphrase comment to make it closer to `Sign` and add cross-reference b/w them: this can make it easier to understand the relationship and differences between the methods

Reply:
Fixed, hand trembled